stable/jaeger-operator/README.md

```diff
@@ -0,0 +1,63 @@
+# jaeger-operator
+
+[jaeger-operator](https://github.com/jaegertracing/jaeger-operator) is a Kubernetes operator.
+
+## Install
+
+```console
+$ helm install stable/jaeger-operator
```

Comment:
Could this be added to `incubator` for the first couple releases?

Reply:
afaik we should not use incubator anymore.

@davidkarlsen @mattfarina @unguiculus is correct? or should I move to incubator?

Reply:
As far as I have understood the incubator is deprecated and we want to go straight for stable - which makes sense - make it proper 1st time and avoid the hassle of promotion.

Reply:
👍